lib/views/layout.erb

```diff
@@ -1,32 +1,57 @@
-<!DOCTYPE html>
-<html>
-<head>
-<body>
-<title>The Gluttany</title>
-
-<h1>The Gluttany</h1>
-
-<p>
-  <% if current_user %>
-    Logged in as <%= current_user.username %>
-    <a href="/auth/logout">Log Out</a></p>
-  <% else %>
-
-    <a href="/auth/login">Log In</a>
-  <% end %>
-
-  <% if flash[:success] %>
-    <div style="color:green;">
-      <%= flash[:success] %>
-    </div>
-  <% end %>
+<!doctype html>
+<html class="no-js" lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Gluttony</title>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/foundation.css">
+    <link rel="stylesheet" href="css/app.css">
+
+  </head>
+
+  <body>
+    <div class="grid-x grid-padding-x">
+      <div class="cell">
+        <center><img src="./img/logo.png" alt="" class=".logo" style="margion: 1000px;"></center>
```

Comment:
using the `center` tag feels so early 2000. Is there no built-in css class for this? Why not create one? ;-)